mathics/core/symbols.py

```diff
@@ -123,7 +146,7 @@ def __pow__(self, other) -> "BaseElement":
 # to be intercombined here.
 
 
-class KeyComparable(object):
+class KeyComparable:
```

Comment:
@TiagoCavalcante a historical note: the reason `object `was there originally is that way back when this code was written (but the change to Python went in around 2.4 or so) there was skill a notion of the the kind of Python lame "class" that Python originally had.  

It always amazed me how long it too Python to fix things that started out doing brokenly but not were broken in other contemporaneous languages.

Back then, in order to get the current style class you had to use this `X(object)` form. 
https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python

At some point, probably Python 3 the non-broken from of class became the default. But I think in Python 2.7 that wasn't the default. 

These are just stories you can tell your grandchildren.

Reply:
Oh - more stories you can tell your grandchildren about the good old days. 

Back then there was no notion of a "folder" or "directory" so all programs had to put in one "file" which was called "a deck of cards".  So Originally Mathics was one file. 

And this was before graph theory was invented and they were still trying to figure out whether all the nodes in a tree should be called "leaves"

Reply:
> @TiagoCavalcante a historical note: the reason `object `was there originally is that way back when this code was written (but the change to Python went in around 2.4 or so) there was still a notion of the the kind of Python lame "class" that Python originally had.
> 
> It always amazed me how long it took Python to fix things that started out doing brokenly but not were broken in other contemporaneous languages.
> 
> Back then, in order to get the current style class you had to use this `X(object)` form. https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python
> 
> At some point, probably Python 3 the non-broken from of class became the default. But I think in Python 2.7 that wasn't the default.
> 
> These are just stories you can tell your grandchildren.

Oh, I didn't know it, I thought that in Python 2 every class needed to extend something.